go/vt/tabletserver/sysloglogger/sysloglogger.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2016, Google Inc. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package sysloglogger
+
+import (
+	"flag"
+	"fmt"
+	log "github.com/golang/glog"
+	"github.com/youtube/vitess/go/vt/servenv"
+	"github.com/youtube/vitess/go/vt/tabletserver"
+	"log/syslog"
+)
+
+// syslogWriter is an interface that wraps syslog.Writer so it can be faked.
+type syslogWriter interface {
+	Info(string) error
+	Close() error
+}
+
+// writer holds a persistent connection to the syslog daemon
+var writer syslogWriter
```

Comment:
I tried adapting the pattern that I see in `event/syslogger/syslogger.go` and it's accompanying unit test suite... holding the syslog connection in a package-level variable that can be replaced with a mock for tests.  However, there are some more unit testing hurdles to overcome, see the comments and notes in the test class below.